app/services/idv/steps/inherited_proofing/verify_info_step.rb

```diff
@@ -25,6 +26,18 @@ def extra_view_variables
         def pii
           flow_session[:pii_from_user]
         end
+
+        private
+
+        def save_proofing_components
+          raise 'current_user is not present' unless current_user.present?
+
```

Comment:
Can this ever happen? If I'm following the logic, we should already be validating current user by the time we reach this logic ([source](https://github.com/18F/identity-idp/blob/397b51f81c27dce33092765877e41b4e99c5e6d3/app/controllers/concerns/idv_session.rb#L6)).

```suggestion
```

Reply:
Correct.